code/src/index.css

```diff
@@ -1,13 +1,148 @@
+/* GLOBAL TARGET */
+
 body {
-  margin: 0;
+  margin: 1em;
   font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
     "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
-}
-
-code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
+  background: linear-gradient(120deg,
+  #FFA8A8,
+  #f8ffba); 
+  background-size: 500%;
+  animation-name: gradient;
+  animation-duration: 3s;
+  animation-iteration-count: infinite;
+  animation-direction: alternate;
+}
+
+@keyframes gradient {
+  from {
+    background-position: 0%;
+  }
+  to {
+    background-position: 100%;
+  }
+}
+
+
+/* STYLING LIST OF THOUGHTS */
+
+.thoughts-container, .form-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.thoughts-cards, form {
+  padding: 20px;
+  border: 1px solid rgb(114, 114, 114);
+  background-color: #fbffd4;
+  width: 300px;
+  box-shadow: 5px 10px #4b433f;
+  margin-bottom: 30px;
+}
+
+.time-and-like-container {
+  display: flex;
+  justify-content: space-between;
+}
+
+.like-container {
+  display: flex;
+  align-items: center;
+}
+
+.thoughts-message {
+  font-weight: 200;
+  font-size: 13px;
+}
+
+.no-like-btn, .like-btn {
+  height: 37px;
+  width: 37px;
+  border-radius: 50%;
+  border: none;
+  background-color: rgb(240, 238, 238);
+  margin-right: 10px;
+  padding-left: 2px;
+  cursor: pointer;
+}
+
+.like-btn {
+  background-color: #FFA8A8;
+}
+
+.like-btn:hover, .no-like-btn:hover {
+  background-color: #fe8282;
+  transform: scale(1.05);
+  transition-timing-function: ease;
+}
```

Comment:
Love the little hover effect on the like buttons! Works really well and gives the user a good experience. Again, great eye for detail that creates a better experience for the user.